web/profiles/custom/os2loop/themes/os2loop_theme/assets/components/pages/document-collection.scss

```diff
@@ -1,5 +1,5 @@
 .os2loop-documents-collection {
 	.tab-content {
-		@extend .block-primary-white;
+		@extend .block-primary-white !optional;
```

Comment:
Why is `!optional` needed here? Are we changing styling extensively to make it work with dompdf?
If that's the case we should clean up the styling rather than using `!optional` to make Sass shut up (https://sass-lang.com/documentation/at-rules/extend/#mandatory-and-optional-extends).

Reply:
Removed